src/Redux/Missions/missions.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable camelcase */
```

Comment:
please remove  eslint disable and follow camelcase. ( your first letter must be small and the first word of seconde word must be small no space ) [look at here](https://www.geeksforgeeks.org/how-to-convert-string-to-camel-case-in-javascript/)